### ISSUE TITLE
Failover

### ISSUE THREAD

**Issue by philiplb** [flynn/flynn#2076]
Hi,
I just tried what happens, if one of my three flynn nodes goes down: The one where the example application is deployed to.
What I somehow expected: flynn moves the application to another node
But I got this message in my browser, even after waiting for a few minutes:

> Service Unavailable

Is this kind of failover somehow available without having the application run on more than one node?

**Comment by josephglanville** [comment]
This is expected to work. The scheduler should realise the node running your job has gone away and start it on another host. The router then should direct requests to this new job.

Could you check if the application appears to be running? (`flynn -a <app_name> ps` is likely to be useful)
It might also be a good idea to run `flynn-host collect-debug-info` on the surviving machines and post the gist URLs here.

**Comment by philiplb** [comment]
It works now.
I think changing the host name from my first flynn node from "flynn" to "flynn0" confused something. So I started over by installing flynn on all three machines, discovery-initialization etc.
I deployed the app, checked it in the browser, shut down the node, waited a bit, checked the browser again and tada! The app is still there, now on a different container.
So it worked as expeced. :)
